touchforms/formplayer/migrations/0001_initial.py

```diff
@@ -22,7 +22,7 @@ class Migration(migrations.Migration):
                 ('session_name', models.CharField(max_length=100)),
                 ('created_date', models.DateTimeField(default=datetime.datetime.utcnow)),
                 ('last_activity_date', models.DateTimeField(null=True)),
-                ('user', models.ForeignKey(to=settings.AUTH_USER_MODEL)),
+                ('user', models.ForeignKey(to=settings.AUTH_USER_MODEL, on_delete=models.CASCADE)),
```

Comment:
The behavior is unchanged, since ```CASCADE``` is the default value for ```on_delete```: https://docs.djangoproject.com/en/1.7/ref/models/fields/#django.db.models.CASCADE

Later when django gets upgraded to 2.0, explicitly setting ```on_delete``` will be required, so this change can stay as is at that time.